README.md

```diff
@@ -1,81 +1,92 @@
-# HMS Ewon Ignition Integration Demo #
+# HMS Ewon® Ignition Demo #
 
 ----------
 
 ## About ##
-HMS Ewon Ignition Integration Demo is a small suite of applications that showcase the features of integration between HMS Networks' Ewon platform, and the Inductive Automation Ignition platform.
+HMS Ewon Ignition Demo is a project to create and maintain a demo that showcases the features of integrating the HMS Networks' Ewon platform and the Inductive Automation Ignition platform.
 
-HMS Ewon Ignition Integration Demo was developed by HMS Networks using the Ewon Java Library. For more information on this demo and developing applications with the Ewon Java Library, please contact us at [us-services@hms.se](mailto:us-services@hms.se "us-services@hms.se") and/or visit the Ewon Developer website at [https://developer.ewon.biz/](https://developer.ewon.biz/ "https://developer.ewon.biz/")
+This project was developed and is maintained by HMS Networks, using the Ewon Java library. For more information about this demo and/or developing applications with the Ewon Java library, please contact us at [us-services@hms.se](mailto:us-services@hms.se "us-services@hms.se"), visit the Ewon Developer website at [https://developer.ewon.biz/](https://developer.ewon.biz/ "https://developer.ewon.biz/")
 
```

Comment:
I think the grammar looks good. You could probably add some sort of table of contents here to jump to headers

Reply:
added